.changesets/feat_garypen_177_coprocessor_metrics.md

```diff
@@ -0,0 +1,16 @@
+### Add coprocessor metrics ([PR #3483](https://github.com/apollographql/router/pull/3483))
+
+Introduces a new metric for the router:
+
+```
+apollo.router.operations.coprocessor
+```
+
+It has two attributes:
+
+```
+coprocessor.stage: string (RouterRequest, RouterResponse, SubgraphRequest, SubgraphResponse)
```

Comment:
Minor nit. Would be nice if these names matched the other use in the metrics naming.
e.g.
`router.request`, `router.response` etc rather than CamelCase.